sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -38,6 +40,9 @@
 # The number of digits to round the price to.
 PRICE_ROUNDING = 5
 
+# The number of retries for the TPU API.
+TPU_RETRY_TIMES = 3
```

Comment:
nit: Let's keep the naming convention the same as other places in our code.
```suggestion
TPU_RETRY_CNT = 3
```